kotsadm/web/src/components/shared/FileTree.jsx

```diff
@@ -19,21 +19,17 @@ export default class FileTree extends React.Component {
   }
 
   getLevel = () => {
-    return this.props.level || 0
+    return this.props.level || 1;
```

Comment:
won't this affect other components using the filetree?

Reply:
updated code